test/vec_notation.lean

```diff
@@ -4,13 +4,17 @@ https://github.com/leanprover-community/mathlib/blob/fee91d74414e681a8b72cb7160e
 -/
 import Mathlib.Data.Fin.VecNotation
 
-/-! These tests are testing `PiFin.toExpr` and fail with
-`local attribute [-instance] PiFin.toExpr` -/
-
 open Lean
 open Lean.Meta
+open Lean.Elab
+open Lean.PrettyPrinter
```

Comment:
These can probably be removed now.

Please make sure to change the PR title to something like `fix: pretty print `vecEmpty` as `![]` even when applied` and update the PR description.

Once you address these things, it's ready to merge. Thanks!

bors d+